features/plugins.feature

```diff
@@ -35,3 +35,22 @@ Feature: Configuring and using plugins
     And I should see "Whatever" in "_site/index.html"
     And the "_site/test.txt" file should exist
     And I should see "this is a test" in "_site/test.txt"
+
+  Scenario: Plugin is enabled
+    Given I have an "index.html" page that contains "{% if site.plugins contains "jekyll_test_plugin" %} plugin loaded {% endif %}"
```

Comment:
Did you test that this works if the tag itself is called (e.g., `{% seo %}`)? IIRC the issue was with liquid parsing the entire file in one pass, rather than respecting `if` logic.